upgrade/scripts/upgrade/util/upgrade-cray-nls.sh

```diff
@@ -50,38 +50,57 @@ function patchKeycloak() {
     --data-urlencode username="$(kubectl get secrets keycloak-master-admin-auth -o jsonpath='{.data.user}' -n services | base64 -d)" \
     --data-urlencode password="$(kubectl get secrets keycloak-master-admin-auth -o jsonpath='{.data.password}' -n services | base64 -d)" \
     https://api-gw-service-nmn.local/keycloak/realms/master/protocol/openid-connect/token | jq -r '.access_token')
```

Comment:
Should also check here for a valid token.

Reply:
throw out bash, problem solved